### PR TITLE
Updated links

### PR DESCRIPTION
**Pull request by ghost**
# Replaced links

## Fixed the Invalid cross reference link.

### CODE REVIEW

File: docs/framework/wcf/system-provided-bindings.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Bindings specify the communication mechanism to use when talking to an endpoint
 |<xref:System.ServiceModel.BasicHttpContextBinding>|Basic Profile 1.1|(None), Transport, Message, Mixed|(None)|(None)|n/a|Text, (MTOM)|Yes<br /><br /> (buffered)|  
 |<xref:System.ServiceModel.NetTcpContextBinding>|.NET|(Transport), Message, None, Mixed|(Transport), Reliable Session, Security Session|(None), Yes|Yes|Binary|Yes<br /><br /> (buffered)|  
 |<xref:System.ServiceModel.WSHttpContextBinding>|WS|Transport, (Message), Mixed|(None), Reliable Session, Security Session|(None), Yes|n/a|Text, (MTOM)|No|  
-|<xref:System.ServiceModel.Channels.UdpBinding>|.NET **Note:**  Interoperability can be achieved by implementing the standard SOAP-over-UDP spec which this binding implements.|(None)|(None)|(None)|n/a|(Text)|No|  
+|<xref:System.ServiceModel.Channels.Binding>|.NET **Note:**  Interoperability can be achieved by implementing the standard SOAP-over-UDP spec which this binding implements.|(None)|(None)|(None)|n/a|(Text)|No|
```
Comment (mairaw) [inline docs/framework/wcf/system-provided-bindings.md]: i think the right link here would be System.ServiceModel.UdpBinding

File: docs/framework/configure-apps/file-schema/wcf/authentication-of-clientcertificate-element.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Specifies authentication behaviors for client certificates used by a service.
 ## Syntax  
   
 ```  
+
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/authentication-of-clientcertificate-element.md]: this change showing up here tells me you were no longer working with the latest from our master branch. do you know how to rebase so you don't reintroduce bugs we already fixed? i can send instructions offline if you don't.

**Review by mairaw — DISMISSED** [review]
Thanks @v-justch. I'll look over more closely tomorrow. Left just some quick comments for now.

File: docs/framework/configure-apps/file-schema/wcf/authentication-of-clientcertificate-element.md (hunk 2 above)
Comment (v-justch) [inline docs/framework/configure-apps/file-schema/wcf/authentication-of-clientcertificate-element.md]: Yes, please. Can you send me instructions on how to rebase. Thanks!

**Comment by mairaw** [comment]
I've rebased this one and redid some of our earlier fixes that rebase didn't bring back. We need to check the fixes now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**